project/Publish.scala

```diff
@@ -50,7 +51,7 @@ object SonatypePublish extends AutoPlugin {
       if (isSnapshot.value) Some("snapshots" at nexus + "content/repositories/snapshots")
       else Some("releases" at nexus + "service/local/staging/deploy/maven2")
     },
-    publish := PgpKeys.publishSigned.value
+    releasePublishArtifactsAction := PgpKeys.publishSigned.value
```

Comment:
https://github.com/lightbend/paradox/blob/04700fda361a17bdc8a12c91b44d951b41a03a2e/build.sbt#L129 

says `^ publish`, so I think you have to change there to `^ publishSigned` instead.

Reply:
thanks Eugene, I amended the commit to include that change